api_tests/test_endpoints.py

```diff
@@ -0,0 +1,33 @@
+import requests
+import json
+
+
+def run_this():
+    url = "http://127.0.0.1:5000/scores"
```

Comment:
This url made need to to change to something like localhost... or 0.0.0.0... for it to work in circleCI. No need to change now but just a heads up for that potential